core/domain/suggestion_services.py

```diff
@@ -0,0 +1,382 @@
+
+
+def get_commit_message_for_suggestion(author_username, commit_message):
+    """Returns a modified commit message for an accepted suggestion.
+
+    Args:
+        author_username: str. Username of the suggestion author.
+        commit_message: str. The original commit message submitted by the
+            suggestion author.
+
+    Returns:
+        str. The modified commit message to be used in the exploration commit
+        logs.
+    """
+    return '%s %s: %s' % (
+        feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX,
+        author_username, commit_message)
+
+
+def accept_suggestion(suggestion, reviewer_id, commit_message):
+    """Accepts the given suggestion after validating it.
+
+    Args:
+        suggestion: Suggestion. The domain object of the suggestion to be
+            accepted.
+        reviewer_id: str. The ID of the reviewer accepting the suggestion.
+        commit_message: str. The commit message.
+
+    Raises:
+        Exception: The suggestion is already handled.
+        Exception: The suggestion is not valid.
+        Exception: The commit message is empty.
+    """
+    if is_suggestion_handled(suggestion):
+        raise Exception('The suggestion has already been accepted/rejected.')
+    if not commit_message or not commit_message.strip():
+        raise Exception('Commit message cannot be empty.')
+
+    try:
+        # If an exception is raised, the suggestion status will be set as
+        # STATUS_INVALID.
+        suggestion.validate()
+    except utils.ValidationError:
+        invalidate_suggestion(suggestion)
+        raise Exception('The given suggestion is not valid.')
+
+    author_name = user_services.get_username(suggestion.author_id)
+    commit_message = get_commit_message_for_suggestion(
+        author_name, commit_message)
+    mark_review_completed(
+        suggestion, suggestion_models.STATUS_ACCEPTED, reviewer_id)
+    suggestion.accept(commit_message)
+
+    feedback_services.create_message(
+        get_thread_id_from_suggestion_id(suggestion.suggestion_id), reviewer_id,
+        feedback_models.STATUS_CHOICES_FIXED, None,
+        'Accepted by %s' % reviewer_id)
+
+
+def reject_suggestion(suggestion, reviewer_id):
+    """Rejects the suggestion.
+
+     Args:
+        suggestion: Suggestion. The domain object of the suggestion to be
+            rejected.
+        reviewer_id: str. The ID of the reviewer rejecting the suggestion.
+
+    Raises:
+        Exception: The suggestion is already handled.
+    """
+    if is_suggestion_handled(suggestion):
+        raise Exception('The suggestion has already been accepted/rejected.')
+
+    mark_review_completed(
+        suggestion, suggestion_models.STATUS_REJECTED, reviewer_id)
+    feedback_services.create_message(
+        get_thread_id_from_suggestion_id(suggestion.suggestion_id), reviewer_id,
+        feedback_models.STATUS_CHOICES_IGNORED, None,
+        'Rejected by %s' % reviewer_id)
```

Comment:
Here, I think the message should definitely be a (mandatory) addition by the reviewer to indicate what the problem is.

Reply:
Added a new parameter as mentioned above. Raised exception if the parameter is None.